### DISCUSSION TITLE
Copy styles.

### DISCUSSION THREAD

**Discussion by assati** [givanz/Vvveb#117] [General]
Hello.
Is it possible to copy styles in vvveb cms?
For example  there are grid of products in Shop and Home (Latest products), and I added styles to ones in Shop and would like the same in Home.

**Comment by givanz** [comment]
Hi

Thanks for the use case description and suggestion.

Currently the css styles generated by the page builder can't be edited from the interface.

It's added in 
```html
<style id="vvvebjs-styles"></style>
```
in the page `<head>` tag.

I will the ability to edit the css from the page builder interface to allow manual tweaking and copy pasting between pages in the next update.

**Comment by assati** [comment]
Thank you.
And is it possible to do like that?:
For example I have header that I want to be the same on every page, and when I edit it on one page, the corrections are automatically applied on others pages too.

**Comment by givanz** [comment]
Header navigation and footer on all pages are linked.

Any change made to header/footer on any page will be applied to all pages.

The only exception is for homepage where the header navigation is not linked with other pages and the changes made on home navigation will not apply on other pages and vice versa.

The navigation on homepage is not linked with other pages because it is different, it is transparent for the hero background to be visible through the navigation.

---

If you want the hero navigation to be linked with all pages you can open code editor and change `data-v-save-global="index.html,.navigation-3"` to `data-v-save-global="blank.html,.navigation-1"` for

```html
<nav class="navigation-3 clearfix fixed-top transparent" data-v-save-global="index.html,.navigation-3">
```

![global-navigation-save](https://github.com/givanz/Vvveb/assets/302738/b3827658-33fa-43ef-9572-a48aac8b3351)
